config/locales/oc.yml

```diff
@@ -272,6 +310,7 @@ oc:
     subscriptions:
       callback_url: URL de rapèl
       confirmed: Confirmat
+      demote: Retrogradar
```

Comment:
Move this line under https://github.com/tootsuite/mastodon/pull/5905/files#diff-7cde7553e8a510f83c4856fb298d9809L64.